templates/index.html

```diff
@@ -1,27 +1,968 @@
+                      class="mdc-text-field__input"
+                      placeholder="Wallet Name"
+                    />
+                    <div class="mdc-line-ripple"></div>
+                  </div>
+                  <div
+                    class="mdc-select mdc-select-box crypto-coin-select"
+                    id="coin-select"
+                  >
+                    <select class="mdc-select__native-control">
+                      <option value="btc" selected>BTC</option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+              <footer class="mdc-dialog__footer">
+                <button
+                  type="button"
+                  class="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--cancel"
+                >
+                  Cancel
+                </button>
+                <button
+                  type="button"
+                  class="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--accept mdc-dialog__action"
+                >
+                  Add
+                </button>
+              </footer>
+            </div>
+            <div class="mdc-dialog__backdrop"></div>
+          </aside>
+        </div>
+      </main>
+    </div>
+
+    <div class="crypto-menu-switches">
+      <div class="crypto-menu-switches--handle">
+        <i class="material-icons">settings</i>
+      </div>
+      <!-- RTL switch -->
+      <div class="switch-wrapper mdc-theme--on-primary">
+        <div class="mdc-switch crypto-rtl-switch-selector">
+          <div class="mdc-switch__track"></div>
+          <div class="mdc-switch__thumb-underlay">
+            <div class="mdc-switch__thumb">
+              <input
+                type="checkbox"
+                id="rtl-switch"
+                class="mdc-switch__native-control"
+                role="switch"
+                aria-checked="false"
+              />
+            </div>
+          </div>
+        </div>
+        <label for="rtl-switch">RTL</label>
+      </div>
+      <!-- theme switch -->
+      <div class="switch-wrapper mdc-theme--on-primary">
+        <div class="mdc-switch crypto-theme-switch-selector">
+          <div class="mdc-switch__track"></div>
+          <div class="mdc-switch__thumb-underlay">
+            <div class="mdc-switch__thumb">
+              <input
+                type="checkbox"
+                id="crypto-theme-switch"
+                class="mdc-switch__native-control"
+                role="switch"
+                aria-checked="false"
+              />
+            </div>
+          </div>
+        </div>
+        <label for="crypto-theme-switch">Switch theme</label>
+      </div>
+    </div>
+
+    <!-- initialise material components js -->
+    <script src="https://code.jquery.com/jquery-1.9.1.min.js"></script>
+    <script src="{{ url_for('static', path='/assets/js/material-components-web.min.js') }}"></script>
```

Comment:
Can we remove `assets/js/material-components-web.min.js` and instead link to the [Material Components CDN](https://material.io/develop/web/getting-started) for this resource?
```html
<link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
<script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
```